probe/host/controls.go

```diff
@@ -0,0 +1,66 @@
+package host
+
+import (
+	"os/exec"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/kr/pty"
+
+	"github.com/weaveworks/scope/common/xfer"
+	"github.com/weaveworks/scope/probe/controls"
+)
+
+// Control IDs used by the host integration.
+const (
+	ExecHost = "host_exec"
+)
+
+// Controls handles controls for a hosts.
+type Controls struct {
+	pipes controls.PipeClient
+}
```

Comment:
I thought about it after you suggested offline but:

* I am planning to extend host controls with logs next, so we will probably need to separate it from the reporter anyhow.
* Now there are architecture specific files for controls.